source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -596,6 +630,8 @@ are not replica set member states at all.
 +-------------------+---------------------------------------------------------------+
 | RSGhost           | "isreplicaset: true" in response.                             |
 +-------------------+---------------------------------------------------------------+
+| LoadBalanced      | "loadBalanced=true" in URI and "serviceId" in response.       |
```

Comment:
Should we remove the "serviceId in response" requirement? There's no monitoring in LB mode and the topology transitions from Unknown to LoadBalanced before any commands are sent to the server, so the server responses are not used to determine SDAM transitions.

Reply:
Removed.